--- xevil-2.02.orig/cmn/intel.cpp	Wed Jan 19 12:54:34 2000
+++ xevil-2.02/cmn/intel.cpp	Mon Jul  2 20:44:30 2001
@@ -1606,7 +1606,7 @@
 Boolean Machine::weapon_current_is_drop_liftable(PhysicalP p) {
   // This is a HACK, we should really make Lifter override 
   // get_weapon_current(), etc.
-  char *dummy;
+  const char *dummy;
   ClassId classId = p->get_weapon_string(dummy);
   if (classId == A_DropLiftable) {
     return True;
--- xevil-2.02.orig/x11/ui.cpp	Wed Jan 19 12:54:14 2000
+++ xevil-2.02/x11/ui.cpp	Mon Jul  2 21:09:11 2001
@@ -1322,7 +1322,7 @@
       
       // Get font.  Use user-specified font if given.
       // regular size font.
-      char *theFont = fontName ? fontName : DEFAULT_FONT_NAME;
+      const char *theFont = fontName ? fontName : DEFAULT_FONT_NAME;
       xvars.font[xvars.dpyMax] = 
         XLoadQueryFont(xvars.dpy[xvars.dpyMax],theFont);
       if (!xvars.font[xvars.dpyMax]) {
