--- xevil-2.02r2/cmn/physical.h.orig	2000-01-19 12:54:08.000000000 +0100
+++ xevil-2.02r2/cmn/physical.h	2003-06-30 22:00:04.000000000 +0200
@@ -93,7 +93,7 @@
    CLASSNAME::CLASSNAME(InStreamP in,const CLASSNAME ## Context &cx, \
                          CLASSNAME ## Xdata &x_data, \
                          WorldP w,LocatorP l) \
-   : PARENT(in,cx. ## parent ## Context,x_data,w,l) 
+   : PARENT(in,cx. parent ## Context,x_data,w,l) 
 
 // Used by internal nodes and leaves.
 #define DEFINE_UPDATE_FROM_STREAM(CLASSNAME,PARENT) \
